src/math/sqrtf.rs

```diff
@@ -29,83 +27,103 @@ pub fn sqrtf(x: f32) -> f32 {
             }
         }
     }
-    let mut z: f32;
-    let sign: i32 = 0x80000000u32 as i32;
-    let mut ix: i32;
-    let mut s: i32;
-    let mut q: i32;
-    let mut m: i32;
-    let mut t: i32;
-    let mut i: i32;
-    let mut r: u32;
+    #[cfg(target_feature = "sse")]
+    {
+        // Note(Lokathor): If compile time settings allow, we just use SSE, since
```

Comment:
I'd remove the "Lokathor" part of the note here, it's not really directed at any one person per se.

Reply:
oh, sure, it was "note from lokathor", in case people had questions later, but i can leave the name off in the future.

Reply:
Comments updated